webpack.config.js

```diff
@@ -2,32 +2,114 @@ const defaultWebpackConfig = require('@cerner/webpack-config-terra');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 const { DefinePlugin } = require('webpack');
 const { merge } = require('webpack-merge');
+const path = require('path');
+const fs = require('fs');
+const TerraDevSite = require('./packages/terra-dev-site/src/webpack/plugin/TerraDevSite');
+
+const html = fs.readFileSync(require.resolve('./tests/terra-dev-site/head.html'), 'utf8');
 
 const terraApplicationConfig = (env = {}) => ({
   entry: {
-    index: './tests/wdio/test-harness/index.jsx',
-  },
-  devServer: {
-    historyApiFallback: true,
-  },
-  output: {
-    publicPath: '/',
+    'terra-application-test/index': './tests/test-harness/index.jsx',
```

Comment:
we host both terra dev site and the terra-application test harness. We're keeping the terra-application test harness so we can test terra-application in true isolation.